cilium-cli/connectivity/check/test.go

```diff
@@ -799,6 +799,17 @@ func (t *Test) NewGenericAction(s Scenario, name string) *Action {
 	return t.NewAction(s, name, nil, nil, features.IPFamilyAny)
 }
 
+// Actions returns a list of failed Actions in the Test.
```

Comment:
nit: looks like this godoc comment was copied from `(*Test).Actions` and should probably be adjusted to document `(*Test).Scenarios` instead.

Reply:
Right you are. I changed my mind mid way on whether to iterate actions or scenarios. Fixed, thanks for pointing that out!